source/material_model/reactive_fluid_transport.cc

```diff
@@ -299,6 +299,12 @@ namespace aspect
                              "permeability $\\phi_0=0.05$. The bulk viscosity additionally "
                              "scales with $\\phi_0/\\phi$. The shear viscosity is read in "
                              "from the base model. Units: dimensionless.");
+          prm.declare_entry ("Minimum compaction viscosity", "1",
+                             Patterns::Double (0),
+                             "Lower cutoff for the compaction viscosity. Units: \\si{\\pascal\\second}.");
+          prm.declare_entry ("Maximum compaction viscosity", "1e50",
+                             Patterns::Double (0),
+                             "Upper cutoff for the compaction viscosity. Units: \\si{\\pascal\\second}.");
```

Comment:
I would change this to have the minimum be zero and the maximum be 
`boost::lexical_cast<std::string>(std::numeric_limits<double>::max())` (you can have a look at Phase transition temperature upper limits in material model utilities.)
This way, you can be sure the default does not change anything.